src/table/table.component.spec.ts

```diff
@@ -69,25 +72,25 @@ describe("Table", () => {
 	it("should call the row filter function", () => {});
 
 	it("should emit a select all event", () => {
-		spyOn(tableInstance.componentInstance.selectAll, "emit");
+		spyOn(tableHeaderInstance.componentInstance.selectAll, "emit");
 
 		let checkbox = fixture.nativeElement.querySelector("th input[type='checkbox']");
 		checkbox.click();
 		fixture.detectChanges();
 
-		expect(tableInstance.componentInstance.selectAll.emit).toHaveBeenCalled();
+		expect(tableHeaderInstance.componentInstance.selectAll.emit).toHaveBeenCalled();
 	});
 
 	it("should emit a deselect all event", () => {
-		spyOn(tableInstance.componentInstance.deselectAll, "emit");
+		spyOn(tableHeaderInstance.componentInstance.deselectAll, "emit");
 
 		let checkbox = fixture.nativeElement.querySelector("th input[type='checkbox']");
 		checkbox.click();
 		fixture.detectChanges();
 		checkbox.click();
 		fixture.detectChanges();
 
-		expect(tableInstance.componentInstance.deselectAll.emit).toHaveBeenCalled();
+		expect(tableHeaderInstance.componentInstance.deselectAll.emit).toHaveBeenCalled();
```

Comment:
So this test should have just worked 😄 we'll need to add another set of tests for just the header 👍